### ISSUE TITLE
`make check` failure

### ISSUE THREAD

**Issue by videlec** [flatsurf/intervalxt#78]
```
cc1plus: warning: ./external/catch2/single_include/catch2/: No such file or directory [-Wmissing-include-dirs]
dynamical_decomposition.test.cc:24:10: fatal error: catch.hpp: No such file or directory
   24 | #include "catch.hpp"
      |          ^~~~~~~~~~~
compilation terminated.
```

**Comment by saraedum** [comment]
Did you `git submodule update --init`?

**Comment by videlec** [comment]
This line is in my `update.sh` script. Let me try again.

**Comment by videlec** [comment]
Something has changed (probably my fault). check runs but fails
```
FAIL: interval_exchange_transformation.py
```
test-suite.log
```
=============================================
   pyintervalxt 0.0.1: test/test-suite.log
=============================================

# TOTAL: 1
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  1
# XPASS: 0
# ERROR: 0

.. contents:: :depth: 2

FAIL: interval_exchange_transformation.py
=========================================

============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-5.3.5, py-1.8.1, pluggy-0.13.1
rootdir: /home/vincent/programming/flatsurf/intervalxt/pyintervalxt/test
collected 2 items

interval_exchange_transformation.py FF                                   [100%]

=================================== FAILURES ===================================
_____________________ test_IntervalExchangeTransformation ______________________

    def test_IntervalExchangeTransformation():
>       iet = intervalxt.IET((18, 3), (1, 0))

interval_exchange_transformation.py:30: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

lengths = (18, 3), permutation = (1, 0)

    def IET(lengths, permutation):
        from cppyy.gbl import std
>       return intervalxt.makeIET(intervalxt.sample.Lengths[int](lengths), permutation)
E       AttributeError: <namespace cppyy.gbl.intervalxt at 0x56346f365bc8> has no attribute 'makeIET'. Full details:
E         type object 'intervalxt' has no attribute 'makeIET'
E         'intervalxt::makeIET' is not a known C++ class
E         'makeIET' is not a known C++ template
E         'makeIET' is not a known C++ enum

../src/pyintervalxt/cppyy_intervalxt.py:60: AttributeError
_________________________________ test_reduce __________________________________

    def test_reduce():
>       iet = intervalxt.IET((18, 3), (1, 0))

interval_exchange_transformation.py:36: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

lengths = (18, 3), permutation = (1, 0)

    def IET(lengths, permutation):
        from cppyy.gbl import std
>       return intervalxt.makeIET(intervalxt.sample.Lengths[int](lengths), permutation)
E       AttributeError: <namespace cppyy.gbl.intervalxt at 0x56346f365bc8> has no attribute 'makeIET'. Full details:
E         type object 'intervalxt' has no attribute 'makeIET'
E         'intervalxt::makeIET' is not a known C++ class
E         'makeIET' is not a known C++ template
E         'makeIET' is not a known C++ enum

../src/pyintervalxt/cppyy_intervalxt.py:60: AttributeError
============================== 2 failed in 0.09s ===============================
FAIL interval_exchange_transformation.py (exit status: 1)
```

**Comment by saraedum** [comment]
So, intervalxt is not installed locally I assume (in particular, there is no old `intervalxt/cppyy.hpp` that is picked up incorrectly?

Which version of cppyy is this?

**Comment by videlec** [comment]
Indeed, after `make install` the `make check` passes.

**Comment by videlec** [comment]
Sorry for the noise.

**Comment by saraedum** [comment]
No, actually not noise ;) It should not pick up the wrong from `make install`.
